tests/ha/check_after_reboot.pm

```diff
@@ -68,6 +68,8 @@ sub run {
 
     # Synchronize all nodes
     barrier_wait("CHECK_AFTER_REBOOT_END_$cluster_name");
+
+    barrier_wait("HAWK_FENCE_$cluster_name") if (check_var('HAWKGUI_TEST_ROLE', 'server'));
```

Comment:
IMHO dual `barrier_wait` is not very useful nor needed here.

Reply:
The second barrier is to signal the hawk_test module that we're done.